event_model/tests/test_em.py

```diff
@@ -388,11 +387,10 @@ def test_document_router_streams_smoke_test(tmp_path):
     start = run_bundle.start_doc
     dr("start", start)
     stream_resource_doc, compose_stream_datum = compose_stream_resource(
-        spec="TIFF_STREAM",
+        mimetype="application/x-hdf5",
```

Comment:
```suggestion
        mimetype="image/tiff",
```

nit but it'd be nice to preserve previous type to show migration path